test/integration/testsuite/prepare.go

```diff
@@ -170,14 +171,14 @@ func runCommandsBeforeFeature(commands string, runner *util.MinishiftRunner) err
 }
 
 // copyOc copies the oc binary contained in directory structure of /<version>/<platform>/oc(.exe)
-// to cache/oc inside of the testDir.
-func copyOc(ocPath string, testDir string) error {
+// to cache/oc inside of the testDefaultHome.
+func copyOc(ocPath string, testDefaultHome string) error {
```

Comment:
@agajdosi Did you verify it locally?

Reply:
@amitkrout yes, tried on Linux, works as expected